interface/patient_file/birthday_alert/birthday.inc.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sharon Cohen<sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+
+
+function display_birthday_alert($deceased_date,$dob,$pid){
+    if($deceased_date > 0 ){
+        return false;
+    }
+    if(date('m-d') >= date('m-d', strtotime($dob))){
+        if(isbirthdayAlertOff($pid)) {
+            return false;
+        }
+    }
+    return true;
+}
```

Comment:
Which code editor are you using? (Will good for long run to get it to support PSR2 standard code styling)